src/features/VisualNovel/pages/CharactersPage/CharactersPage.tsx

```diff
@@ -59,6 +60,21 @@ export const CharactersPage: VFC = () => {
     return groupedCharacters;
   };
 
+  const [canFit4OrMore, canFitOnly3, canFitOnly2] = useMediaQuery(
+    ['(min-width: 1564px)', '(min-width: 1280px)', '(min-width: 1024px)'],
+  );
```

Comment:
Bad and implicit naming, what if we would want to use media query hook in this component for purposes other than calculating column amount?

Reply:
The sizes are specific for calculating column amount and shouldn't be used anywhere else.